app/konnect/runtime-manager/runtime-instances/custom-dp-labels.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Using custom data plane labels
+content_type: how-to
+---
+
+You can set [labels](/konnect/reference/labels/) on each runtime instance (data plane) to provide custom context information. 
+
+Labels are commonly used for metadata information. Set anything that you need to identify your runtime instances -- deployment type, region, size, the team that the instance belongs to, the purpose it serves, and much more. 
+
+Using labels makes it easier to manage runtime instances at scale, since you're able to identify all of this information at a glance. For example, debugging and troubleshooting become much easier when you're able to attribute a runtime instance to a specific region or team.
```

Comment:
```suggestion
Labels are commonly used for metadata information. Set anything that you need to identify your runtime instances -- deployment type, region, size, the team that the instance belongs to, the purpose it serves, and much more. 

Using labels makes it easier to manage runtime instances at scale since you're able to identify all of this information at a glance. For example, debugging and troubleshooting become much easier when you're able to attribute a runtime instance to a specific region or team.
```